test/data/expected/41-date-literal-03.json5

```diff
@@ -0,0 +1,619 @@
+                                                         operator: '=',
+                                                         value: {
+                                                            type: 'dateLiteral',
+                                                            value: 'THIS_FISCAL_YEAR'
+                                                         }
+                                                      }
+                                                   },
+                                                   right: {
+                                                      type: 'ComparisonCondition',
+                                                      field: {
+                                                         type: 'FieldReference',
+                                                         path: ['CreadedDate']
+                                                      },
+                                                      operator: '=',
+                                                      value: {
+                                                         type: 'dateLiteral',
+                                                         value: 'LAST_FISCAL_YEAR'
+                                                      }
+                                                   }
+                                                },
+                                                right: {
+                                                   type: 'ComparisonCondition',
+                                                   field: {
+                                                      type: 'FieldReference',
+                                                      path: ['CreadedDate']
+                                                   },
+                                                   operator: '=',
+                                                   value: {
+                                                      type: 'dateLiteral',
+                                                      value: 'NEXT_FISCAL_YEAR'
+                                                   }
+                                                }
+                                             },
+                                             right: {
+                                                type: 'ComparisonCondition',
+                                                field: {
+                                                   type: 'FieldReference',
+                                                   path: ['CreadedDate']
+                                                },
+                                                operator: '=',
+                                                value: {
+                                                   type: 'dateLiteral',
+                                                   value: 'LAST_N_DAYS:1',
+                                                   variable: '1'
+                                                }
```

Comment:
I think the consumer program wants the date literal type ("LAST_N_DAYS") rather than the raw value with argument specified ("LAST_N_DAYS:1").

Reply:
```js
{
  type: 'dateLiteral',
  value: 'LAST_N_DAYS',
  variable: 1
}
```

Reply:
@stomita - I have made the requested changes.

- [x] Removed `:` from value
- [x] Renamed `variable` to `argument` (not sure why I could not think of `argument` as the name the first time ;)
- [x] Changed `argument` data type from string to integer
- [x] Updated unit tests